src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.html

```diff
@@ -52,11 +52,9 @@
 <ng-template #statusColor
              let-value="value">
   <span *ngFor="let state of value; last as last">
-    <span [class.text-success]="'up' === state || 'in' === state"
-          [class.text-warning]="'down' === state || 'out' === state">
-      {{ state }}</span>
-    <span *ngIf="!last">, </span>
-    <!-- Has to be on the same line to prevent a space between state and comma. -->
+    <span *ngIf="['up', 'in'].includes(state)" class="label label-success">{{ state }}</span>
```

Comment:
Wouldn't it be better to use ngClass here?

Reply:
ngClass refers to some properties of the component class, which we do not have here. This is not a single state of the component but the state of muItiple OSDs which are iterated. I could, of course, use ngClass here, but that'd result in having all states of all OSDs as an object property of the component class, which would not simplify but complicate this implementation. Unless I'm missing something?

Reply:
Ok